### DISCUSSION TITLE
ScriptReference/EditorGUI.Foldout

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#33] [Notes]
# ScriptReference/EditorGUI.Foldout

https://docs.unity3d.com/ScriptReference/EditorGUI.Foldout.html

**Comment by vertxxyz** [comment]
You can style a [Toggle](https://docs.unity3d.com/ScriptReference/EditorGUI.Toggle.html) like a Foldout by providing [`EditorStyles.foldout`](https://docs.unity3d.com/ScriptReference/EditorStyles-foldout.html) to the Toggle and it will function the same but in some cases (like when not provided a label) the Toggle will function where the Foldout may not.
